web-console/future-openshift/custom-catalogs/custom-catalogs.md

```diff
@@ -0,0 +1,65 @@
+# Adding custom catalogs to OperatorHub
+
+Admins can create custom catalogs in order to give admin console users the ability to add operators from other sources, beyond the default OperatorHub catalog sources.
+
+## Global Configuration
+
+OperatorHub sources are found through: Administation > cluster settings > global configuration > OperatorHub / sources tab.
+
+![global config](img/operatorhub-custom-catalog1.png)
+
+![OperatorHub config](img/operatorhub-custom-catalog2.png)
+
+## Sources
+
+In the OperatorHub configuration, users can see the current default sources that are feeding into OperatorHub. These sources populate OperatorHub and keep installed operators updated.
+
+![OperatorHub sources default](img/operatorhub-custom-catalog3.png)
+
+If users "create catalog source" they will see a form to fill out details.
+
+![OperatorHub catalog source form](img/operatorhub-custom-catalog4.png)
+
+After the form is filled out and users select "create", they will see the newly created catalog source in the list.
+
+![OperatorHub catalog source in list](img/operatorhub-custom-catalog5.png)
+
+Once a new catalog source is created, admin users can see the operators from the catalog source from within OperatorHub.
```

Comment:
I would still prefer to not call the provider type **Custom** but rather the display name of the `CatalogSource`. Using multiple custom catalogs this custom category could get crowded quite quickly.

Reply:
Updated designs to reflect this.